modules/distribution-zones/src/integrationTest/java/org/apache/ignite/internal/distributionzones/ItIgniteDistributionZoneManagerNodeRestartTest.java

```diff
@@ -297,7 +301,8 @@ private PartialNode startPartialNode(int idx) {
         );
 
         for (IgniteComponent component : otherComponents) {
-            component.start();
+            // TODO: had to start without waiting as the test otherwise fails with timeout.
```

Comment:
The problems is that in the common node start you have to call `metaStorageMgr.notifyRevisionUpdateListenerOnStart()`, so some components (like DZM could proceed with its logic, because the wait for some VersionedValues)

```
        CompletableFuture<Void> startupConfigurationUpdate = notifyConfigurationListeners();
        CompletableFuture<Void> startupRevisionUpdate = metaStorageMgr.notifyRevisionUpdateListenerOnStart();

        return CompletableFuture.allOf(startupConfigurationUpdate, startupRevisionUpdate, startFuture)
```

but in this partial node you wait for node to be started without calling `notifyRevisionUpdateListenerOnStart`, so it can't start. At lease this is the root cause for `ItIgniteDistributionZoneManagerNodeRestartTest`. I bet the problem is the same for `ItIgniteNodeRestartTest`

Reply:
I've created a tickrt